### PR TITLE
feat: support jackson config

### PR DESCRIPTION
**Pull request by Lo1nt**
### Motivation:

support jackson config by env and application config

### Modification:

Describe the idea and modifications you've done.

### Result:

Fixes #<GitHub issue number>. 

If there is no issue then describe the changes introduced by this PR.

### CODE REVIEW

File: codec/codec-jackson/src/main/java/com/alipay/sofa/rpc/codec/jackson/JacksonSerializer.java [hunk 1]
```diff
@@ -326,4 +374,41 @@ private void decodeSofaResponse(AbstractByteBuf data, SofaResponse sofaResponse,
             sofaResponse.setAppResponse(result);
         }
     }
+
+    @VisibleForTesting
+    protected ObjectMapper getMapper() {
+        return this.mapper;
+    }
+
+    public static ConfigKey<String> JACKSON_SER_FEATURE_ENABLE_LIST  = ConfigKey
```
Comment (EvenLjj) [inline codec/codec-jackson/src/main/java/com/alipay/sofa/rpc/codec/jackson/JacksonSerializer.java:383]: 成员变量最好挪到最上面

File: codec/codec-jackson/src/main/java/com/alipay/sofa/rpc/codec/jackson/JacksonSerializer.java [hunk 2]
```diff
@@ -326,4 +374,41 @@ private void decodeSofaResponse(AbstractByteBuf data, SofaResponse sofaResponse,
...
+                                                                             false,
+                                                                             "希望被设置为关闭的Serialize FEATURE",
+                                                                             new String[] { "sofa_rpc_codec_jackson_serialize_feature_disable_list" });
+
+    public static ConfigKey<String> JACKSON_DES_FEATURE_ENABLE_LIST  = ConfigKey
+                                                                         .build(
+                                                                             "sofa.rpc.codec.jackson.deserialize.feature.enable.list",
+                                                                             "",
+                                                                             false,
+                                                                             "希望被设置为开启的Deserialize EATURE",
```
Comment (EvenLjj) [inline codec/codec-jackson/src/main/java/com/alipay/sofa/rpc/codec/jackson/JacksonSerializer.java:404]: typo error `EATURE`

File: codec/codec-jackson/src/main/java/com/alipay/sofa/rpc/codec/jackson/JacksonSerializer.java [hunk 3]
```diff
@@ -74,29 +80,71 @@ public class JacksonSerializer extends AbstractSerializer {
...
+     * 获取用户配置的 feature, 应该是逗号分割的 string
+     * 根据 String 获取对应的 DeserializationFeature, 并配置到 object mapper
+     *
+     * @param objectMapper
+     * @param key
+     * @param enable
+     * @param featureSet
+     */
+    private void processDeserializeFeatures(ObjectMapper objectMapper,ConfigKey<String> key, boolean enable, Set<String> featureSet) {
+        String serFeatureList = SofaConfigs.getOrDefault(key);
```
Comment (EvenLjj) [inline codec/codec-jackson/src/main/java/com/alipay/sofa/rpc/codec/jackson/JacksonSerializer.java:146]: typo error `serFeatureList`

File: codec/codec-jackson/src/main/java/com/alipay/sofa/rpc/codec/jackson/JacksonSerializer.java [hunk 4]
```diff
@@ -74,29 +80,71 @@ public class JacksonSerializer extends AbstractSerializer {
...
+     * 获取用户配置的 feature, 应该是逗号分割的 string
+     * 根据 String 获取对应的 SerializationFeature, 并配置到 object mapper
+     *
+     * @param objectMapper
+     * @param key
+     * @param enable
+     * @param featureSet
+     */
+    private void processSerializeFeatures(ObjectMapper objectMapper, ConfigKey<String> key, boolean enable, Set<String> featureSet) {
+        String serFeatureList = SofaConfigs.getOrDefault(key);
```
Comment (EvenLjj) [inline codec/codec-jackson/src/main/java/com/alipay/sofa/rpc/codec/jackson/JacksonSerializer.java:132]: 这里最好加下非空校验，虽然default值为“”。

File: codec/codec-jackson/src/main/java/com/alipay/sofa/rpc/codec/jackson/JacksonSerializer.java [hunk 5]
```diff
@@ -74,29 +80,71 @@ public class JacksonSerializer extends AbstractSerializer {
 
     public JacksonSerializer() {
 
+        Set<String> serFeatures = Arrays.stream(SerializationFeature.values()).map(Enum::name).collect(Collectors.toSet());
```
Comment (EvenLjj) [inline codec/codec-jackson/src/main/java/com/alipay/sofa/rpc/codec/jackson/JacksonSerializer.java:83]: 说明一下以后就是两种方式：第一种是-Dsofa.rpc.codec.jackson.SerializationFeature.FAIL_ON_EMPTY_BEANS=true，第二种方式是-Dsofa.rpc.codec.jackson.serialize.feature.enable.list=FAIL_ON_EMPTY_BEANS，推荐使用第二种方式。

**Review by EvenLjj — APPROVED** [review]

**Review by chuailiwu — APPROVED** [review]
lgtm

**Pull request merged**